server/database/db_dummy_data.sql

```diff
@@ -0,0 +1,15 @@
+BEGIN;
```

Comment:
For the sake of clarity, I would probably move this file into the test folder (and possibly create another folder inside test for the database...). I think it's better to keep the folders clean and put all the dummy data needed with the tests 😄

Reply:
@MynahMarie, Although it is called dummy data it isn't just used for testing. Because we are only doing the customer part of the project and not the cook side we need some cook data to be manually entered in to the DB. This data will be used for development throughout the project. It would only be removed if we went beyond what is scoped. Happy to move it in to the test file if you think that is where this would belong.